Source/Core/VideoBackends/Software/SWVertexLoader.cpp

```diff
@@ -173,13 +173,16 @@ void SWVertexLoader::LoadVertex()
 
 	// convert the vertex from the gc format to the videocommon (hardware optimized) format
 	u8* old = g_video_buffer_read_ptr;
-	m_CurrentLoader->RunVertices(
+	bool skip = !m_CurrentLoader->RunVertices(
 		m_primitiveType, 1,
```

Comment:
This really makes it look like RunVertices returns a boolean... please make this a little more explicit.

Otherwise LGTM.